### ISSUE TITLE
training on coco

### ISSUE THREAD

**Issue by hao522** [amdegroot/ssd.pytorch#122]
Has anyone tried to train on coco? I can't the AP(0.23) from the paper, is there anything special should be changed when dataset is coco?

**Comment by ellisbrown** [comment]
We are working to get training on coco working right now, you can see our changes on the [COCO branch](https://github.com/amdegroot/ssd.pytorch/tree/coco).  We hope to merge these changes into master soon, once we have confirmed expected results from the paper

**Comment by ztianlin** [comment]
@ellisbrown hi~could you please show your training log with COCO? (loss with iterations, how much time, final evaluation results...)
